api/v1alpha1/zz_generated.chaosmesh.go

```diff
@@ -62,6 +62,7 @@ func (in *AwsChaos) GetChaos() *ChaosInstance {
 		StartTime: in.CreationTimestamp.Time,
 		Action:    "",
 		UID:       string(in.UID),
+                Status:    in.Status.ChaosStatus,
```

Comment:
There seem to be extra spaces here?

Reply:
It seems that it's a github bug? Because if you look at this file, you will find that the indentation is normal.

Reply:
Maybe you need to format it